prog/dftb+/lib_dftb/scc.F90

```diff
@@ -875,7 +890,8 @@ subroutine setShiftPerAtom(this, shift)
 
     @:ASSERT(this%tInitialised)
     @:ASSERT(size(shift) == size(this%shiftPerAtom,dim=1))
-    
+
+    this%hasExternalShifts = .true.
```

Comment:
I would rather use something like `@:ASSERT(this%hasExternalShifts)` than setting the variable here.